notebooks/metatranscriptomics_pipeline_iSeqnorm.ipynb

```diff
@@ -0,0 +1,1477 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sns.boxplot(x='Project Plate',y='Sample DNA Concentration',data=plate_df,fliersize=0)\n",
+    "sns.stripplot(x='Project Plate',y='Sample DNA Concentration',data=plate_df,color='grey',alpha=0.5)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for plate in plate_df['Project Plate'].unique():\n",
+    "    plate_needs_dilution = requires_dilution(plate_df.loc[plate_df['Project Plate']==plate,\n",
+    "                          ['Sample DNA Concentration']], threshold=40)\n",
+    "    if plate_needs_dilution:\n",
+    "        #Dilution, 1:dilution factor\n",
+    "        #sample : total volume (as a factor)\n",
+    "        for dilution_factor in range(2,6):\n",
+    "            dilute_ = requires_dilution(plate_df.loc[plate_df['Project Plate']==plate,\n",
+    "                              ['Sample DNA Concentration']]/dilution_factor, threshold=40)\n",
+    "            if dilute_:\n",
+    "                continue\n",
+    "            else:\n",
+    "                print(f'You  need to do a a 1:{dilution_factor} dilution for {plate}')\n",
+    "                break\n",
+    "    else:\n",
+    "        print(f'You do not need a dilution for {plate}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Post DNAse QC (Optional but highly encouraged)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dnase_dna_concs_fp = './test_data/Quant/MiniPico/2024_NPH_Plates_3-6_Post_DNase_gDNA_Quant.txt'\n",
+    "\n",
+    "if not os.path.isfile(sample_dna_concs_fp):\n",
+    "    print(\"Problem! %s is not a path to a valid file\" % file)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Read in the DNA concentration output file**. It should be a .txt file and should look something like this:\n",
+    "    \n",
+    "```\n",
+    "##BLOCKS= 1\r\n",
+    "Group: Unknowns\r\n",
+    "Sample\tWells\tRFU_Values\tConcentration\tMean_Conc\tSD\tCV\tDilution\tAdjConc\t\r\n",
+    "01\tA1\t16352.000\t0.434\t0.434\t0.000\t0.0\t\t\t\r\n",
+    "02\tC1\t26061758.000\t37.588\t37.588\t0.000\t0.0\t\t\t\r\n",
+    "03\tE1\t1483519.000\t2.527\t2.527\t0.000\t0.0\t\t\t\r\n",
+    "04\tG1\t3015184.000\t4.712\t4.712\t0.000\t0.0\t\t\t\r\n",
+    "05\tI1\t16574.000\t0.434\t0.434\t0.000\t0.0\t\t\t\r\n",
+    "06\tK1\t14988.000\t0.432\t0.432\t0.000\t0.0\t\t\t\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dnase_dna_concs = read_pico_csv(dnase_dna_concs_fp, plate_reader='SpectraMax_i3x',conc_col_name='Post DNAse DNA Concentration')\n",
+    "\n",
+    "plate_df = pd.merge(plate_df, dnase_dna_concs, on='Well')\n",
+    "\n",
+    "well_col = 'Well'\n",
```

Comment:
Again, I don't think this should be (re)-defined here if it is not being used in this cell.